modules/generator/registry/config.go

```diff
@@ -15,6 +15,9 @@ type Config struct {
 	// Defaults to 15m.
 	StaleDuration time.Duration `yaml:"stale_duration"`
 
+	// EnableStaleMarkers controls whether stale markers should be sent.
+	EnableStaleMarkers bool `yaml:"enable_stale_markers"`
```

Comment:
is there any reason to make this configurable? if this is correct behavior why don't we just do it.

going to summon @mattdurham who likely knows far more about staleness markers and when/if they are worth sending than myself.

Reply:
Perhaps an ignorant question, are the same series being generated from the same instance always? Side question out of the way, to my knowledge its always 5m.

Reply:
> is there any reason to make this configurable? if this is correct behavior why don't we just do it.
I didn't want to automatically change default behavior w/o providing a way to change it back. No real reason here besides "just in case".  I can get rid of this config.

> are the same series being generated from the same instance always?
The same series might be generated by different instances however they will always be differentiated by the `__metrics_gen_instance` label, so mimir ingesters should see them as different series iiuc.

Reply:
For what its worth Alloy generally hard codes 5m, otel makes it configurable. For max compatibility configuration is reasonable but a comment that 5m is the accepted default that is almost never changed might go a long way.

Reply:
One thing to note here is that there are default adaptive metrics rules in place to aggregate series on the `__metrics_gen_instance` label and I'm not sure what impact that might have.

Reply:
Thanks @mattdurham. Setting the `stale_duration` to 5m makes sense to me. The probably is configurable. 

Do you think it makes sense to make sending stale markers configurable or rather we should just always send them when cleaning up stale series?

Reply:
Off the top of my head I cant think of a downside to sending staleness marker.